packages/gatsby/src/utils/dev-ssr/render-dev-html.ts

```diff
@@ -3,15 +3,17 @@ import fs from "fs-extra"
 import nodePath from "path"
 import report from "gatsby-cli/lib/reporter"
 import { isCI } from "gatsby-core-utils"
-import { Stats } from "webpack"
+import type { Request } from "express"
 import { ROUTES_DIRECTORY } from "../../constants"
 import { startListener } from "../../bootstrap/requires-writer"
 import { findPageByPath } from "../find-page-by-path"
 import { getPageData as getPageDataExperimental } from "../get-page-data"
 import { getDevSSRWebpack } from "../../commands/build-html"
-import { emitter, GatsbyReduxStore } from "../../redux"
+import { GatsbyReduxStore } from "../../redux"
 import { IGatsbyPage } from "../../redux/types"
-
+import { getServerData, IServerData } from "../get-server-data"
+import { getPageMode } from "../page-mode"
+import { parseError } from "./render-dev-html-child"
```

Comment:
this probably should be moved to some common module and not imported from worker / child process entry